pkg/build/controller/policy/policy.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	buildapi "github.com/openshift/origin/pkg/build/api"
 	buildclient "github.com/openshift/origin/pkg/build/client"
+	buildlister "github.com/openshift/origin/pkg/build/generated/listers/build/internalversion"
```

Comment:
The run policies then need to be instantiated with a buildLister created from a client and not the informer then:
https://github.com/openshift/origin/pull/14679/files#diff-4349d2367514c13a1d5c76dc4debf58fR126